modules/data.atmosphere/R/tdm_generate_subdaily_models.R

```diff
@@ -0,0 +1,226 @@
+##' Create statistical models to predict subdaily meteorology from daily means
```

Comment:
It should be fine for downscaling 6hr reanalysis. The only thing that might not make sense are the names, (we have lag.day, next.day, etc.) but these are just the previous and next values, so as long as the dates are built properly it should be fine. I'll change this "daily" part